neuralmonkey/train.py

```diff
@@ -34,8 +35,13 @@ def create_config() -> Configuration:
     config.add_argument("evaluation")
     config.add_argument("runners")
     config.add_argument("test_datasets", required=False, default=[])
-    config.add_argument("logging_period", required=False, default=20)
-    config.add_argument("validation_period", required=False, default=500)
+    # a little "hackish" checking
+    config.add_argument("logging_period",
+                        cond=lambda x: int(re.sub("[^0-9-]", "", str(x))) > 0,
+                        required=False, default=20)
+    config.add_argument("validation_period",
+                        cond=lambda x: int(re.sub("[^0-9-]", "", str(x))) > 0,
+                        required=False, default=500)
```

Comment:
K čemu je tahle kontrola dobrá? Pak se to stejně zkontroluje v training loopu. Jestli je nutnost kontroly tady, tak bych zavolal `learning_utils._resolve_period`, ať je aspoň důsledná.

Reply:
Souhlasim, rad tu kontrolu odsud vyhodim. Prisel s ni Jindra H.